src/Phinx/Config/Config.php

```diff
@@ -495,6 +495,7 @@ protected function parseAgnosticDsn(array $options): array
      * @param mixed $value Value
      * @return void
      */
+    #[\ReturnTypeWillChange]
     public function offsetSet($id, $value): void
```

Comment:
I don't think this one will change.

Reply:
Yeah, total brain fart on doing this, only `configGet` needs it 🤦